src/main/java/com/lorepo/icplayer/Icplayer.gwt.xml

```diff
@@ -17,7 +17,7 @@
     <script src="build/dist/libs.min.js"/>
     <!--Tiny mce have to be loaded separatly, due to URL finding of themes, if we change the location it will wrongly download themes-->
     <script src="libs/tinymce.min.js"/>
-    <!--<script src="http://code.responsivevoice.org/responsivevoice.js"></script>-->
+    <script src="https://code.responsivevoice.org/responsivevoice.js?key=B3vo0UDz"></script>
```

Comment:
responsive voice to płatna biblioteka. Nie wiem, czy to klucz kupiony, czy wersji demonstracyjnej, ale niezależnie od tego pod żadnym pozorem nie może się tu znaleźć. 

Według mnie biblioteka powinna być dodawana przez templatę w mA/mC (o czym wspominałem w tym komentarzu: https://learneticsa.assembla.com/spaces/d8V_EykXmr4BCyacwqjQXA/tickets/9081/details?comment=1706988686 , choć nie wiem czy coś w tym kierunku jest robione), bo to tam będzie pewnie przechowywana informacja, czy responsive voice klient wykupił. Ten element script można ewentualnie dodać do templaty w wersji testowej na sztywno, żeby umożliwić testowanie. W player'ze należy się oczywiście zabezpieczyć przed sytuacją, gdzie tej biblioteki nie ma, żeby nic nie wybuchło.